QModManager/Checks/PirateCheck.cs

```diff
@@ -29,20 +31,20 @@ internal static void PirateDetected()
 
         internal static void IsPirate(string folder)
         {
-            string steamDll = Path.Combine(folder, "steam_api64.dll");
+            var steamDll = Path.Combine(folder, "steam_api64.dll");
             if (File.Exists(steamDll))
             {
-                FileInfo fileInfo = new FileInfo(steamDll);
-
-                if (fileInfo.Length > 220000)
+                FileInfo dllInfo = new FileInfo(steamDll);
+                if (dllInfo.Length > FileSize)
                 {
-                    PirateDetected();
-                    return;
+                    FileChecking(folder);
                 }
             }
-
-            foreach (string file in CrackedFiles)
-            {
+        }
+        
+        private static void FileChecking(string folder)
+        {
+            foreach (var file in CrackedFiles) {
                 if (File.Exists(Path.Combine(folder, file)))
                 {
                     PirateDetected();
```

Comment:
I would propose that this detection be changed to a new log line stating something like:
"Warning!! Primary crack detection passed but known cracked files found. Please verify ownership before assisting user."